src/main/java/ch/njol/skript/expressions/ExprHoverList.java

```diff
@@ -82,13 +63,19 @@ public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelaye
 
 	@Override
 	@Nullable
-	public String[] get(Event e) {
-		if (!(e instanceof PaperServerListPingEvent))
+	public String[] get(Event event) {
+		if (!(event instanceof PaperServerListPingEvent))
```

Comment:
You can use pattern matching for instanceof since we now support Java 17
*And in all the other places*

Reply:
for 2.9, 1.13-1.16 is still on java 11 right?